src/Microsoft.AspNetCore.WebUtilities/project.json

```diff
@@ -20,6 +20,7 @@
   },
   "dependencies": {
     "Microsoft.Extensions.Primitives": "1.1.0-*",
+    "Microsoft.Net.Http.Headers": "1.1.0-*",
```

Comment:
There should be a few packages in this repo that depend on WebUtilities and Headers where you can now remove Headers.